apache/couchdb-fauxton
# Update fauxton/notifications to use redux

## Overview

Update fauxton/notifications to use redux

## Testing recommendations

Global Notifications
- Try to create a database name `db%`. The error notification should be displayed.
- Create a database with a valid name. The success notification is displayed.
  - Verify that clicking the notification close button works

Notification Center
- Open the Notification Center (bell icon on the top right corner)
- All previous notifications are displayed.
- Clicking the filter buttons at the top ('All', 'Success', 'Error', 'Info') and verify the list is updated correctly.
- Click the 'Clear All' button at the bottom and verify the list is cleared.
- Click the Close (X) button at the top right to close the Notification Center

## Checklist
